Assets/PhotoCapture.cs

```diff
@@ -94,6 +96,7 @@ private void Update()
     {
         if (UnityEngine.Input.GetMouseButtonDown(0))
         {
+            soundFX.Play();
```

Comment:
I cannot hear the sound, I do see that soundFX has something assigned but no audio happens when I take a photo